libraries/AP_HAL_Linux/RCInput_UDP_Protocol.h

```diff
@@ -1,11 +1,19 @@
 #pragma once
 
 #define RCINPUT_UDP_NUM_CHANNELS 8
-#define RCINPUT_UDP_VERSION 2
+#define RCINPUT_UDP_NUM_CHANNELS_V3 16
 
 struct __attribute__((packed)) rc_udp_packet {
```

Comment:
Can this structure now be removed?

Reply:
This header is kind of an API. Older clients, may be using it, so I didn't want to break them